setup/gcp/installer/util/utils.py

```diff
@@ -147,32 +147,32 @@ def format_resource_id(resource_type, resource_id):
     return '%s/%s' % (resource_type, resource_id)
 
 
-def format_service_acct_id(prefix, modifier, timestamp, project_id):
+def format_service_acct_id(prefix, modifier, installation_type, project_id):
     """Format the service account ids.
 
     Args:
-        prefix (str): The prefix of the account id
-        modifier (str): Access level of the account
-        timestamp (str): Timestamp of the class
-        project_id (str): Id of the project on GCP
+        prefix (str): The prefix of the account id.
+        modifier (str): Access level of the account.
+        installation_type (str): Type of the installation (client/server).
+        project_id (str): Id of the project on GCP.
 
     Returns:
-        str: Service account id
+        str: Service account id.
     """
 
     return full_service_acct_email(
         constants.SERVICE_ACCT_FMT.format(
-            prefix, modifier, timestamp), project_id)
+            prefix, modifier, installation_type), project_id)
```

Comment:
Thanks for making this change!  Just a qq: will we ever have a need for a ```timestamp``` in the service account name?  e.g. someone installing 2 forseti instances into the same project, and thus need the timestamp to differentiate the 2 instances?  What exactly is the character limitation again?

Reply:
It terms out that service account name (up to 100 characters) can be duplicated and service account id (up to 30 characters) has to be unique. I have separated them out in the new commit, can you please take a look at the new changes?